### PR TITLE
*: update binary package for 6.2.0

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: binary-package.md [hunk 1]
```diff
@@ -64,6 +64,12 @@ TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB
 | reparo |  |
 | arbiter |  |
 | mydumper | 从 v6.0.0 起新增 |
+| server-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
+| grafana-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
+| alertmanager-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
+| prometheus-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
+| blackbox_exporter-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
```
Comment (qqqdan) [inline binary-package.md:71]:
```suggestion
| blackbox_exporter-v0.21.1-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
```

**Review by qqqdan — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]

File: binary-package.md [hunk 2]
```diff
@@ -64,6 +64,12 @@ TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB
 | reparo |  |
 | arbiter |  |
 | mydumper | 从 v6.0.0 起新增 |
+| server-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
+| grafana-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
```
Comment (ran-huang) [inline binary-package.md:68]: This is already listed in TiDB-community-server table

File: binary-package.md [hunk 3]
```diff
@@ -64,6 +64,12 @@ TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB
...
+| alertmanager-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
```
Comment (ran-huang) [inline binary-package.md:69]: ditto

File: binary-package.md [hunk 4]
```diff
@@ -64,6 +64,12 @@ TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB
...
+| prometheus-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
```
Comment (ran-huang) [inline binary-package.md:70]: ditto

File: binary-package.md [hunk 5]
```diff
@@ -64,6 +64,12 @@ TiDB 提供两个二进制离线包：`TiDB-community-server` 软件包和 `TiDB
...
+| blackbox_exporter-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
+| node_exporter-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
```
Comment (ran-huang) [inline binary-package.md:72]:
```suggestion
| grafana-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
| alertmanager-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
| prometheus-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
| blackbox_exporter-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
| node_exporter-{version}-linux-amd64.tar.gz | 从 v6.2.0 起新增 |
```
These items are already in TiDB-community-server. Please

Comment (shichun-0415) [reply binary-package.md:72]: According to [‍‬⁠⁣⁡⁤⁠⁡⁠⁡​‍⁢⁠‍⁢⁢‌‌⁣‬⁢‬‬​‌⁢​‬⁣⁣﻿⁣⁤⁠⁢⁡‬‍⁤⁣‌⁣‍⁡​⁣‌﻿‌TiDB 交付物组件梳理](https://pingcap.feishu.cn/base/bascnk1aTRZX0FNtBj3o31ktdyf?table=tbli47NRTY18ykai&view=vewRuj5zF9), starting from 6.2.0, these items are also added to the toolkit package.
image

Comment (ran-huang) [reply binary-package.md:72]: LGTM

Comment (shichun-0415) [reply binary-package.md:72]: As to the introduction version mentioned in the file, I have confirmed with Zhang Yue, 6.2.0 is earlier than 6.1.1. Therefore, we should change the version to v6.2.0.

Comment (ran-huang) [reply binary-package.md:72]: 6.2.0 for release-6.2, 6.1.1 for release-6.1?

Comment (shichun-0415) [reply binary-package.md:72]: Agree

**Review by ran-huang — APPROVED** [review]

**Pull request merged**